governance/votes/Executive Vote - March 11, 2022.md

```diff
@@ -0,0 +1,116 @@
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes and additions** will occur within the Maker Protocol:
+- A total of 2,750 DAI will be transferred to two addresses, detailed below, as part of the [Immunefi Bug Bounty Program](https://mips.makerdao.com/mips/details/MIP64).
+- Curve stETH-ETH LP tokens will be onboarded as a new collateral to the Maker Protocol as the CRVV1ETHSTETH-A vault type, parameters listed below.
+- A new rate limiter, detailed below, will be added to the flapper contract, and flap (surplus) auctions will be re-activated.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Immunefi Bug Bounty Payouts
+
+As part of the [Immunefi Bug Bounty Program](https://mips.makerdao.com/mips/details/MIP64), a total of 2,750 will be transferred as follows if this executive proposal passes:
+
+* **2,500 DAI** will be transferred to [0x3C32F2ca11D92a7093d1F237161C1fB692F6a8eA](https://etherscan.io/address/0x3C32F2ca11D92a7093d1F237161C1fB692F6a8eA).
+* **250 DAI** will be transferred to [0x2BC5fFc5De1a83a9e4cDDfA138bAEd516D70414b](https://etherscan.io/address/0x2BC5fFc5De1a83a9e4cDDfA138bAEd516D70414b).
+
+For further details, please see the relevant forum thread [here](https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-5565/13545).
+
+### Onboarding CRVV1ETHSTETH-A
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/Qmek9vzo), the CRVV1ETHSTETH-A vault type will be onboarded with the following parameters if this executive proposal passes. Please note the updated parameters proposed by the Risk Core Unit (RISK-001) in this forum [post](https://forum.makerdao.com/t/curvelp-steth-eth-parameters-adjustment/13767).
+
+-- Initial Collateral Parameters --
```

Comment:
For these I usually * anything that differs from the poll, and call out why they are different in the detail section.

Reply:
OK I've changed it, does it seem ok to you?